src/components/subscriptions/licenses/LicenseManagementTable/index.jsx

```diff
@@ -150,6 +150,31 @@ const LicenseManagementTable = () => {
     [currentPage],
   );
 
+  const filtersAreAppliedToColumns = (
```

Comment:
can probably do something fancy here to not need these 2 helper functions (when passing stuff into the bulkaction)